pattern_import_export/models/ir_exports_select_tab.py

```diff
@@ -16,20 +16,52 @@ class IrExportsSelectTab(models.Model):
     field_id = fields.Many2one("ir.model.fields", string="Field", required=True)
 
     @api.multi
-    def _generate_additional_sheet(self, book, bold):
+    def _get_header(self):
+        """
+        Get the header
+        @return: list of string
+        """
+        self.ensure_one()
+        return [self.field_id.name]
+
+    @api.multi
+    def _get_records_to_export(self):
+        """
+        Get recordset to export
+        @return: recordset
+        """
+        self.ensure_one()
+        field = self.field_id.name
+        model = self.model_id.model
+        domain = []
+        if self.domain:
+            domain = ast.literal_eval(self.domain)
+        return self.env[model].read_group(domain, [field], [field], orderby=field)
+
+    @api.multi
+    def _get_data_to_export(self):
+        """
+        Iterator who built data dict record by record
+        @return: dict
+        """
+        for record in self._get_records_to_export():
+            data = {}
+            for header in self._get_header():
+                data.update({header: record[header]})
+            yield data
+
+    @api.multi
+    def _generate_additional_sheet(self, book, style):
```

Comment:
following code is specific to the implementation of xlsx

Reply:
Yes I forgot to remove it in this PR but I remove it into the next one (#9 ). So to avoid conflicts with the future merge, I propose to let it and it'll be deleted when the next one is merged.